archive/archive.go

```diff
@@ -446,23 +436,23 @@ func extractPackageTarball(b *ui.Task, r io.Reader, dest string, strip int) erro
 	return nil
 }
 
-func extractDebianPackage(b *ui.Task, r io.Reader, dest string, pkg *manifest.Package) error {
+func extractDebianPackage(b *ui.Task, r io.Reader, dest string, pkg *manifest.Package) (func() error, error) {
```

Comment:
Maybe I am missing something, but isn't this return value always ignored? Is it needed?